care/emr/reports/context_builder/builders/allergy.py

```
@@ -0,0 +1,179 @@
+    "unable_to_assess": "Unable to Assess",
+}
+
+ALLERGY_INTOLERANCE_TYPE_DISPLAY = {
+    "allergy": "Allergy",
+    "intolerance": "Intolerance",
+}
+
+
+class AllergyContextBuilder(QuerysetContextBuilder):
+    model = AllergyIntolerance
+    depends_on = ["encounter_id"]
```

Ruff (0.14.6)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)